File: java assignments/Day02assignment/src/com/sonata/day02assignment/Tshirt.java
package com.sonata.day02assignment;
class shirt
{
	String color;
	String material;
	String design;
	shirt(String a , String b ,String c)
	{
		this.color =a;
		this.material =b;
		this.design =c;
		
	}
	void display()
	{
		System.out.println(color);
		System.out.println(material);
		System.out.println(design);
	}
}


public class Tshirt {

	public static void main(String[] args) {
		shirt small = new shirt("yellow","cotton","v shape");
		shirt large = new shirt("red","cotton","round neck");
		shirt xl = new shirt("orange","slick","v shape");
		small.display();
		large.display();
		xl.display();
		
		

	}

}

File: typescript examples/assignments/Tshirt.ts
class Tshirt{
    color:string;
    material:string;
    design:string;
    size:string;

    constructor(siz:string,col:string,  matr:string,des:string){
        this.color = col;
        this.design = des;
        this.material = matr;
        this.size = siz;
    }
    display(){
        console.log("Size - " + this.size 
                + "\n Material - " + this.material + 
                    "\n Design - "  + this.design  
                    +"\n Color - " + this.color);
    }
}

var t1 = new Tshirt("large","rose","silk","full hands");
var t2 = new Tshirt("small","red","cotton","V-neck");
var t3 = new Tshirt("x_l","orange","cotton","round-neck");

t1.display();
t2.display();
t3.display();